Source/SwiftLintFramework/Models/Command.swift

```diff
@@ -81,7 +81,10 @@ public struct Command: Equatable {
 
         let ruleTexts = scanner.string.bridge().substring(
             from: scanner.scanLocation + 1
-        ).components(separatedBy: .whitespaces)
+        ).components(separatedBy: .whitespaces).filter {
+            let component = $0.trimmingCharacters(in: .whitespaces)
+            return !component.isEmpty && component != "*/"
```

Comment:
not a big fan of this, but I thought it would be better to keep supporting it even if it was unintentional